src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -1163,8 +1165,8 @@ type TaskSeq =
 
     /// <summary>
     /// Applies the function <paramref name="folder" /> to each element in the task sequence, threading an accumulator
-    /// argument of type <typeref name="'State" /> through the computation.  If the input function is <paramref name="f" /> and the elements are <paramref name="i0...iN" />
-    /// then computes <paramref name="f (... (f s i0)...) iN" />.
+    /// argument of type <typeref name="'State" /> through the computation.  If the input function is <code>f</code> and the elements are <code>i0...iN</code>
```

Comment:
This doesn't get rendered with coloring, so let's stick with the hacky way, until I get time to fix the xml doc parsing code of the F# compiler itself.

```suggestion
    /// argument of type <typeref name="'State" /> through the computation. If the input function is <paramref name="f" /> and the elements are <paramref name="i0...iN" />
```

Reply:
yeah, `<code>` also forces a line break - `<c>` is my goto in Equinox
The problem is that Rider checks paramrefs, so abusing them means a LOT of noise

Reply:
👍  now paramref as discussed
(also I meant `<c>` anyway)